### PR TITLE
feat(global): DOMA-7425 B2C-miniapps MVP 📱👨‍💻😎

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

**Review by AleX83Xpert — APPROVED** [review]

**Review by sitozzz — APPROVED** [review]

File: packages/clients/address-service-client/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function createTestInstance (existingItem = null) {
  * @returns {MockedAddressServiceClient|AddressServiceClient}
  */
 function createInstance (testItem) {
-    return conf.NODE_ENV === 'test' || get(conf, 'ADDRESS_SERVICE_CLIENT_MODE') === 'fake'
+    return conf.NODE_ENV === 'test' || get(conf, 'ADDRESS_SERVICE_CLIENT_MODE', 'fake') === 'fake'
```
Comment (Alllex202) [inline packages/clients/address-service-client/index.js:34]: Are you sure these changes are needed?

File: apps/dev-portal/lib/gql/miniapp.graphql [hunk 2]
```diff
@@ -0,0 +1,73 @@
+query allApps($creator: UserWhereInput!, $first: Int!) {
+    b2c: allB2CApps(sortBy: createdAt_DESC, where: { createdBy: $creator }, first: $first) {
+        id
+        name
```
Comment (Alllex202) [inline apps/dev-portal/lib/gql/miniapp.graphql:4]: Just a question, is this autogen or not?

File: apps/dev-portal/lib/auth.tsx [hunk 3]
```diff
@@ -121,11 +152,12 @@ export function extractAuthHeadersFromRequest (req: { cookies: Partial<Record<st
...
-export async function prefetchAuth (client: ApolloClient<NormalizedCacheObject>, opts: { headers?: HeadersType } = {}): Promise<void> {
-    await client.query({
+export async function prefetchAuth (client: ApolloClient<NormalizedCacheObject>, opts: { headers?: HeadersType } = {}): Promise<AuthenticatedUserType> {
+    const response = await client.query<AuthenticatedUserQuery>({
         query: AuthenticatedUserDocument,
         context: {
             headers: opts.headers,
         },
     })
+    return response.data.authenticatedUser
```
Comment (Alllex202) [inline apps/dev-portal/lib/auth.tsx:162]: Should we extract `authenticatedUser` safely?

File: apps/dev-portal/domains/miniapp/hooks/useFileValidator.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    const intl = useIntl()
+    const ErrorTitle = intl.formatMessage({ id: 'global.errors.fileUpload.title' })
+    const fileSizeFormatter = useFileSizeFormatter()
+
+    return useCallback((file: RcFile) => {
+        return new Promise((resolve) => {
+            if (opts.restrictMimeTypes && !opts.restrictMimeTypes.includes(file.type)) {
+                const ErrorDescription = intl.formatMessage({ id: 'global.errors.fileUpload.wrongType.description' })
+                notification.error({ message: ErrorTitle, description: ErrorDescription })
+                return resolve(Upload.LIST_IGNORE)
```
Comment (Alllex202) [inline apps/dev-portal/domains/miniapp/hooks/useFileValidator.ts:29]:
```suggestion
                return reject(Upload.LIST_IGNORE)
```

File: apps/dev-portal/lib/gql/miniapp.graphql (hunk 2 above)
Comment (SavelevMatthew) [reply apps/dev-portal/lib/gql/miniapp.graphql:4]: This one is not. `.ts` one is

File: apps/dev-portal/pages/apps/b2c/[id].tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+                </Row>
+            </BaseLayout>
+        </>
+    )
+}
+
+export default AppSettingsPage
+
+export const getServerSideProps: GetServerSideProps = async ({ req, query }) => {
+    const { id, section, p } = query
```
[the remark this answers is not in the document]
Comment (SavelevMatthew) [reply apps/dev-portal/pages/apps/b2c/[id].tsx:107]: I'll check it :)))0)

File: apps/dev-api/domains/miniapp/plugins/exportable.js
Comment (Alllex202) [inline apps/dev-api/domains/miniapp/plugins/exportable.js:1]: How about moving this plugin to `packages/keystone/plugins`?

File: apps/dev-portal/pages/apps/b2c/[id].tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (SavelevMatthew) [reply apps/dev-portal/pages/apps/b2c/[id].tsx:107]: `p` is a shorthand for `page` (similar to `q` is shorthand for search/question). Here it's used to receive a tab name (or section, or page...)

File: packages/clients/address-service-client/index.js (hunk 1 above)
Comment (SavelevMatthew) [reply packages/clients/address-service-client/index.js]: Added backward-compatible change in another PR (outdated)

File: apps/dev-portal/pages/apps/b2c/[id].tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (Alllex202) [reply apps/dev-portal/pages/apps/b2c/[id].tsx:107]: Ok, why then `section` and not `s`? :)))

**Pull request merged**